templates/drupal8/files/.platform.template.yaml

```diff
@@ -23,7 +23,11 @@ info:
   # The human-readable name of the template.
   name: Drupal 8
   # Human-readable descriptive text for the template. Supports limited HTML.
-  description: Drupal is a highly-customizable free software PHP CMS that allows you to easily organize, manage and publish your content.
+  description: |
+    This template builds Drupal 8 using the [Drupal Composer project](https://github.com/drupal-composer/drupal-project)
+    for better flexibility.  It also includes configuration to use Redis for caching, although that must be enabled
+    post-install in `.platform.app.yaml`.
```

Comment:
This is by design a copy of the opening of the README.

I don't know if Markdown is supported here or not.  @Kazanir?

I don't know if repeating "This template builds" at the start of every description is useful, but grammatically I don't know how to avoid it.